### DISCUSSION TITLE
Sidebar or Sheet?

### DISCUSSION THREAD

**Discussion by hsavit1** [shadcn-ui/ui#5446] [Q&A]
When would you use either? Is sidebar a replacement for sheet?

**Comment by Steveb599** [comment]
Sheet is for mobile imo

**Reply by hsavit1** [reply]
I dont understand, im supposed to use one for desktop and the other for mobile? that doesnt make sense

**Comment by discoverlance-com** [comment]
I think ideally, you use sidebar for your dashboards. Sheets can also be used for cases like mobile view navigations or any other like even allowing user to view notifications.

Sheet is not static as it requires a trigger to open so it might not be ideal for dashboards where you need the navigation or the content to stick around on larger screens.  It can be used if you build your own custom sidebar  for desktop and then you can use a sheet to display navigation for mobile view.

Sidebars can be used independently without needing to include a sheet because it supports customizations that allows it to adapt to the screen size whether desktop, tablet or mobile.

By using sidebar for dashboards, you get a composable sidebar that you can customize to your liking for both mobile and desktop use and you can continue to use sheet for other aspects like viewing notifications, mobile navigation on your websites with your typical home/about/contact pages and such relevant use cases.
